.github/workflows/ind.yaml

```diff
@@ -29,15 +24,11 @@ jobs:
         elif [[ "${{ github.ref }}" == *"master" ]]; then
           echo "VERSION=latest" >> $GITHUB_ENV
         fi
-    - name: Get owner and repo
-      run: |
-        ownerrepo=${{ github.repository }}
-        owner=$(echo $ownerrepo | cut -f1 -d/)
-        if [[ "$owner" == "iter8-tools" ]]; then
-          owner=iter8
-        fi
-        echo "OWNER=$owner" >> $GITHUB_ENV
-        echo "REPO=ind" >> $GITHUB_ENV
```

Comment:
aren't OWNER and REPO still needed? I still see them in the `tags` argument to build-push-action

Reply:
I think this may have been a mistake.